File: app/src/main/java/net/programmierecke/radiodroid2/station/live/metadata/lastfm/data/Album.java

package net.programmierecke.radiodroid2.station.live.metadata.lastfm.data;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Album {

    @SerializedName("artist")
    @Expose
    private String artist;
    @SerializedName("title")
    @Expose
    private String title;
    @SerializedName("url")
    @Expose
    private String url;
    @SerializedName("image")
    @Expose
    private List<Image> image = null;

    public String getArtist() {
        return artist;
    }

    public void setArtist(String artist) {
        this.artist = artist;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public List<Image> getImage() {
        return image;
    }

    public void setImage(List<Image> image) {
        this.image = image;
    }

}

File: app/src/main/java/net/programmierecke/radiodroid2/station/live/metadata/lastfm/data/Album.kt
package net.programmierecke.radiodroid2.station.live.metadata.lastfm.data

import com.google.gson.annotations.SerializedName

data class Album(
        @SerializedName("artist")
        val artist: String? = null,

        @SerializedName("title")
        val title: String? = null,

        @SerializedName("url")
        val url: String? = null,

        @SerializedName("image")
        val image: List<Image>? = null
)